containers/daap-python-base/src/var/task/data_platform_paths.py

```diff
@@ -36,6 +37,29 @@
 MAX_IDENTIFIER_LENGTH = 255
 
 
+class MetadataTypes(Enum):
```

Comment:
Suggestion: Maybe something like `SpecificationType` or `JsonSchemaName` would be a more helpful name here - I get it now that I know what it's doing, but it's so easy to confuse the specification paths with the actual schema or metadata paths.

Reply:
yeah, am trying to avoid using `metadata` where possible, so i like it